src/components/networks/network.py

```diff
@@ -719,7 +732,23 @@ def calculate_max_capex():
         # CAPEX auxilliary (used to calculate theoretical CAPEX)
         # For new technologies, this is equal to actual CAPEX
         # For existing technologies it is used to calculate fixed OPEX
-        b_arc.var_capex_aux = Var(bounds=calculate_max_capex())
+        if hasattr(b_arc, "var_capex_aux"):
```

Comment:
Can we write this if statement differently? The problem is: if you for some other reason (because we have another problem) have var_capex_aux already, it just executes these lines, which we dont want. Maybe we can even set the bounds outside of the network class (in the monte carlo function), because that is the only time we actually need it?

Reply:
Having the bounds and constraint generation outside of the network class in the Monte Carlo class is what I had initially :') I think your intention with doing it in the network file was to keep the code cleaner, which I agree with. The problem was however that it wasn't working since the var_capex was overwritten due to this function. We could split the definition of parameters, variables and constraints just like with the technologies?